testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5095,7 +5141,7 @@ QUnit.module("performance", () => {
         $(".dx-list-select-all-checkbox").trigger("dxclick");
 
         // assert
-        assert.equal(keyGetterCounter, 613, "key getter call count");
+        assert.equal(keyGetterCounter, 614, "key getter call count");
```

Comment:
```suggestion
        assert.equal(keyGetterCounter, 613, "key getter call count");
```
If we can use `list.option("selectedItemKeys")` only...